react/features/speaker-stats/components/web/SpeakerStatsLabels.tsx

```diff
@@ -69,6 +69,7 @@ const SpeakerStatsLabels = (props: Props) => {
             {
                 props.showFaceExpressions
                 && <div className = { `expressions ${classes.emojis}` }>
+                    {/* @ts-ignore */}
```

Comment:
What error is produced here?

Reply:
`Element[] is not valid JSX`. I fixed it by embedding the code directly here